docs/source/fundamentals.md

```diff
@@ -101,29 +99,155 @@ Once a certain degree of interoperability has been reached, other interesting co
   Being able to know the semantic meaning of an instance and hence its connection to other concepts
   is one of the principles of SimPhoNy. For that, ontologies play a major role.
```

Comment:
1. I would wrap "and hence its connection to other concepts" with commas. 
2. Petty comment: I think the style of "For that, ontologies play a major role." is a bit rough. Suggestion: replace "for that" with "for achieving this goal"